src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,69 @@
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+
 import './CurrentUser.scss';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+import { getUser } from '../../api/todos';
+import { Loader } from '../Loader';
+import { User } from '../types';
+
+type Props = {
+  selectedUserId: number,
+  setSelectedUserId: (userId: number) => void,
```

Comment:
```suggestion
  setSelectedUserId: Dispatch<SetStateAction<number>>,
```
import such types from react and use them for all state update functions